pydra/engine/helpers_file.py

```diff
@@ -517,3 +512,14 @@ def is_local_file(f):
     from .specs import File
 
     return f.type is File and "container_path" not in f.metadata
+
+
+def is_existed_file(f):
```

Comment:
What is the difference between this and `os.path.exists`?

Reply:
one problem with `os.path.exists` is that gives `True` for `os.path.exists(4)`

Reply:
Gotcha. Maybe just:

```Python
try:
    return Path(f).exists()
except TypeError:
    return False
```

Then we don't have to worry about specifically supporting `py.path` or any other `PathLike`?